<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@865.v43e78cc44e0d">
  <node class="org.jenkinsci.plugins.workflow.graph.FlowEndNode" plugin="workflow-api@1275.veb_e0969ddf9e">
    <parentIds>
      <string>16</string>
    </parentIds>
    <id>17</id>
    <startId>2</startId>
    <result>
      <name>ABORTED</name>
      <ordinal>4</ordinal>
      <color>ABORTED</color>
      <completeBuild>false</completeBuild>
    </result>
  </node>
  <actions>
    <wf.a.ErrorAction plugin="workflow-api@1275.veb_e0969ddf9e">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@639.v6eca_cd8c04a_a_">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.interrupt(CpsFlowExecution.java:1183)</trace>
          <trace>org.jenkinsci.plugins.workflow.job.WorkflowRun$2.lambda$interrupt$0(WorkflowRun.java:397)</trace>
          <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:67)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions>
          <wf.a.ErrorAction_-ErrorId plugin="workflow-api@1275.veb_e0969ddf9e">
            <stackTrace/>
            <suppressedExceptions class="empty-list"/>
            <uuid>af381068-571c-467d-9d46-6c979fe1c23d</uuid>
          </wf.a.ErrorAction_-ErrorId>
          <java.lang.NullPointerException>
            <stackTrace>
              <trace>org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:2848)</trace>
              <trace>org.jenkinsci.plugins.gitclient.CliGitAPIImpl$9.execute(CliGitAPIImpl.java:3152)</trace>
              <trace>hudson.plugins.git.GitSCM.checkout(GitSCM.java:1355)</trace>
              <trace>org.jenkinsci.plugins.workflow.steps.scm.SCMStep.checkout(SCMStep.java:129)</trace>
              <trace>org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:97)</trace>
              <trace>org.jenkinsci.plugins.workflow.steps.scm.SCMStep$StepExecutionImpl.run(SCMStep.java:84)</trace>
              <trace>org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)</trace>
              <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
              <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
              <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
              <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
              <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
            </stackTrace>
            <suppressedExceptions class="empty-list"/>
          </java.lang.NullPointerException>
        </suppressedExceptions>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <jenkins.model.CauseOfInterruption_-UserInterruption>
              <user>kjb95</user>
            </jenkins.model.CauseOfInterruption_-UserInterruption>
          </a>
        </causes>
        <actualInterruption>true</actualInterruption>
      </error>
    </wf.a.ErrorAction>
    <wf.a.TimingAction plugin="workflow-api@1275.veb_e0969ddf9e">
      <startTime>1694601170526</startTime>
    </wf.a.TimingAction>
  </actions>
</Tag>